only apply domain eps for floating and complex types

pmeier opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #96334
* #96333
* #96124
* __->__ #97010
* #96332
* #97017
* #96125
* #96331

As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

pmeier committed:
only apply domain eps for floating and complex types

[ghstack-poisoned]

pmeier committed:
Update on "only apply domain eps for floating and complex types"


As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

[ghstack-poisoned]

pmeier committed:
Update on "only apply domain eps for floating and complex types"


As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

[ghstack-poisoned]

mruberry reviewed (approved):
<img src="https://media4.giphy.com/media/26n7bwoaNiNvjEYbm/giphy.gif"/>

pmeier committed:
Update on "only apply domain eps for floating and complex types"


As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

[ghstack-poisoned]

pmeier commented:
Seems like a real failure: https://hud.pytorch.org/pr/97010#12167701793

pmeier committed:
Update on "only apply domain eps for floating and complex types"


As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

[ghstack-poisoned]

pmeier committed:
Update on "only apply domain eps for floating and complex types"


As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

[ghstack-poisoned]

pmeier committed:
Update on "only apply domain eps for floating and complex types"


As discussed in https://github.com/pytorch/pytorch/pull/96124#issuecomment-1471973352.

[ghstack-poisoned]
